athena-oracle/athena-oracle.yaml

```diff
@@ -58,6 +61,8 @@ Resources:
           spill_bucket: !Ref SpillBucket
           spill_prefix: !Ref SpillPrefix
           default: !Ref DefaultConnectionString
+          ssl: !Ref Ssl
+          secret_name: !Ref SecretNamePrefix
```

Comment:
Shouldn't need this once the code is updated to make use of the existing jdbc credential provider